SymPEP-0001.md

```diff
@@ -0,0 +1,222 @@
+- To decide on the process by which a change will be implemented, for example,
+  whether the change should be implemented in several stages, whether
+  intermediate releases are required for things like deprecations, and so on.
+- To document the above discussions and decisions for future reference.
+- To document the motivations for a change from the perspective of end users
+  who may be affected by it. Here "end users" means both users of the SymPy
+  library, as well as developers working on parts of SymPy itself which may be
+  affected by the change.
+
+Importantly, **a SymPEP is not documentation** for the proposed change. End
+user documentation should be included with the implementation of the feature
+in the corresponding SymPy documentation. This also means that other
+documentation should not cross-reference a SymPEP as if it were documentation
+for a feature. Even technical discussion of a feature should be documented
+separately from a SymPEP. The reason is that SymPEPs will necessarily include
+details that are irrelevant to the final implementation, such as discussions
+of alternate implementations which were rejected and discussions of
+implementation of the change.
+
+## SymPEP Workflow
+
+Discussions around a proposed change may begin informally on the mailing list
+or SymPy issue tracker. However, once it is decided that the formal SymPEP
+process is desired for a change, the discussion should move to the
+[SymPEP repository](https://github.com/sympy/SymPEPs) on GitHub.
+
+Every SymPEP must have a champion. This is a person who is responsible for
+writing the SymPEP, leads the discussion around it, and tries to create
+consensus around it.
+
+The author of a SymPEP should fork the repository and create a pull request
+with a new SymPEP document based on the [SymPEP template](SymPEP-template).
+The SymPEP document may be named `SymPEP-XXXX.md` until a number is assigned.
+One of the core SymPy developers will then assign a number to the SymPEP, in
+which case `XXXX` should be replaced with the number with leading 0s. The
+person who assigns the number should also update the
+[README](https://github.com/sympy/SymPEPs/blob/main/README.md) of the main
+[SymPEP repository](https://github.com/sympy/SymPEPs) to list that number.
+Numbers should be assigned to SymPEPs as soon as it is determined that they
+are a legitimate proposal. If a SymPEP ends up being rejected or postponed, it
+keeps its number, as rejection or postponement status is still a discussion
+that should be documented in the SymPEP. SymPEP numbers should generally be
+assigned in increasing numeric order.
+
+Discussion on the SymPEP should then continue on the SymPEP pull request.
+Discussions may also take place in other places, such as [GitHub
+discussions](https://github.com/sympy/SymPEPs/discussions) or the [mailing
+list](http://groups.google.com/group/sympy). All discussions should be
+cross-referenced in the "Discussions" section of the SymPEP document.
```

Comment:
I've noticed in #3, #4, and #5 that it isn't quite clear if the SYMPEP is ready for review, versus being a WIP draft that someone is working on. I suggest we use Github's "Draft Pull Request" feature to indicate WIP and once the PEP is complete that status is changed. This would help us know which PEPs we should focus more immediate attention.